src/openrct2/core/StringBuilder.h

```diff
@@ -24,92 +24,54 @@ class StringBuilder final
 {
```

Comment:
Look into the headers that are included pretty sure iterator and algorithm can be moved to the cpp file. Not sure if they are even used any more.